app/views/layouts/ats/_navbar.html.slim

```diff
@@ -12,28 +16,22 @@ nav.navbar.navbar-expand-lg.align-items-center.fixed-top.header.gap-2
     #navbarNav.collapse.navbar-collapse.gap-lg-2.order-5.order-lg-1
       ul.navbar-nav.text-nowrap.gap-1.gap-lg-0.mb-1.mb-lg-0
         - if allowed_to?(:index?, with: ATS::DashboardPolicy)
-          li.nav-item
-            = link_to t("core.dashboard"), root_path,
-                      class: "nav-link#{' nav-link-selected' if @nav_item == :dashboard}"
+          li.nav-item class=("active" if @nav_item == :dashboard)
+            = link_to t("core.dashboard"), root_path, class: "nav-link"
         - if allowed_to?(:index?, with: ATS::TaskPolicy)
-          li.nav-item
-            = link_to ats_tasks_path,
-              class: "nav-link#{' nav-link-selected' if @nav_item == :tasks} d-inline-flex align-items-center " \
-                     "gap-2 pt-md-2"
+          li.nav-item class=("active" if @nav_item == :tasks)
+            = link_to ats_tasks_path, class: "nav-link gap-1 pt-md-2 justify-content-between"
```

Comment:
Why is there a top padding of 8px on the desktop?

Reply:
no idea, removing